File: Dream.WebApp/app/classes/models/marketRateEnvironmentModel.ts
import { MarketRateDataModel } from './marketRateDataModel';
import { RateCurveDataModel } from './rateCurveDataModel';

export class MarketRateEnvironmentModel {
    IsModified: boolean;
    MarketRateDataModel: MarketRateDataModel;
    RateCurveDataModel: RateCurveDataModel;
}

File: Dream.WebApp/Models/MarketRateEnvironmentModel.cs
namespace Dream.WebApp.Models
{
    public class MarketRateEnvironmentModel
    {
        public bool IsModified { get; set; }
        public MarketRateDataModel MarketRateDataModel { get; set; }
        public RateCurveDataModel RateCurveDataModel { get; set; }
    }
}